platform/configure/config.mdx

```diff
@@ -68,9 +72,9 @@ config:
       redirectURI: https://my-vcluster-platform-host.com/auth/github/callback
 ```
 
-Then use helm to apply this configuration and update `$PLATFORM_VERSION` with a valid vCluster Platform version.
+Then use Helm to apply this configuration and update `$PLATFORM_VERSION` with a valid platform version.
 
-```
+```bash title="Apply configuration using Helm"
```

Comment:
This command is outdated and should be updated to match the install docs in variables and styling:

Match the styling in here: https://www.vcluster.com/docs/platform/install/helm#installing-the-platform-helm-chart